.gitignore

```diff
@@ -11,7 +11,6 @@ Dockerfile
 
 # test_files
 dry_run.py
-test_*
```

Comment:
let's not remove this, kept it to ignore test scripts

Reply:
It skips test files for my branch as well, will update it once we are done with all other changes